File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--binding-data-to-controls-on-a-word-actions-pane_3.vb
    Shared Sub SetHeadings(ByVal tblCell As Word.Cell, ByVal text As String)

        With tblCell.Range
            .Text = text
            .Font.Bold = True
            .ParagraphFormat.Alignment = Word.WdParagraphAlignment.wdAlignParagraphCenter
        End With
    End Sub

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--binding-data-to-controls-on-a-word-actions-pane_3.cs
        static void SetHeadings(Microsoft.Office.Interop.Word.Cell tblCell, string text)
        {
            tblCell.Range.Text = text;
            tblCell.Range.Font.Bold = 1;
            tblCell.Range.ParagraphFormat.Alignment = 
                Microsoft.Office.Interop.Word.WdParagraphAlignment.wdAlignParagraphCenter;
        }